src/sentry/static/sentry/app/utils/theme.tsx

```diff
@@ -542,12 +542,12 @@ export const darkTheme = {
   button: generateButtonTheme(darkAliases),
 } as const;
 
-export type Theme = typeof theme | typeof darkTheme;
+export type Theme = typeof lightTheme | typeof darkTheme;
 export type Color = keyof typeof colors;
 export type IconSize = keyof typeof iconSizes;
 export type Aliases = typeof aliases;
 
-export default theme;
+export default commonTheme;
```

Comment:
should we really export it as a default? wouldn't it be better if it was a named export?

Reply:
I think semantically speaking, most users directly importing from theme.tsx should be using commonTheme. Pragmatically speaking, many current users of theme.tsx do in fact only require commonTheme (e.g. for breakpoints), and in my opinion, removing the default export causes unnecessary code churn.